src/clj/web/data.clj

```diff
@@ -1,11 +1,20 @@
 (ns web.data
+  (:refer-clojure :exclude [sort find])
   (:require [web.db :refer [db object-id]]
             [web.utils :refer [response]]
             [monger.collection :as mc]
             [monger.result :refer [acknowledged?]]
+            [monger.query :refer :all]
```

Comment:
Would it be better to not exclude functions from Clojure core but use a prefix?l here? What do you think?

Reply:
Agreed. Just copying from examples on the web.